src/assets/variables.scss

```diff
@@ -0,0 +1,16 @@
+@import "./fonts";
+
+//fonts
+$text-login-regular: normal 18px "Montserrat";
+$text-login-bold: 700 36px "Montserrat";
+$text-error-message-semibold: 600 14px "Montserrat";
+$text-login-button-bold: 700 18px "Montserrat";
```

Comment:
Why do you write login styles in variables?
If it's needed for many components, change name just to 'text-regular' and so on; if it's not declare it in needed component